app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,13 +52,41 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="assignedContact"
+              placeholder="Select contact"
+              options={CONTACTS_DATA.map(({ label, value }) => ({
+                label,
+                value,
+              }))}
+            />
+            <Select
+              isClearable
+              isMulti
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tags"
+              placeholder="Select tag"
+              options={TAGS_DATA.map(({ label, value }) => ({
+                label,
+                value,
```

Comment:
I don't understand the point of mapping here? Can you please explain? Thanks.

Reply:
I am sorry. This is a careless mistake on my end. Since the `buildSelectOptions` method is returning the `TAGS_DATA` in the correct format, there is no need for the `map` function here. I will change this to `options={TAGS_DATA}` and then, this should work.